services/ui-src/src/components/reports/ReportPage.test.tsx

```diff
@@ -56,77 +27,97 @@ jest.mock("utils", () => ({
   },
 }));
 
-const standardReportPageComponent = (
+const mockUpdateReport = jest.fn();
+const mockUpdateReportData = jest.fn();
+const mockedReportContext = {
+  ...mockReportContext,
+  updateReport: mockUpdateReport,
+  updateReportData: mockUpdateReportData,
+};
+
+const ReportPageComponent_StandardForm = (
```

Comment:
Could we use the abstracted code you wrote here? Specifically these lines and use them as the mocked report context?

```
export const mockReportMethods = {
  setReport: jest.fn(),
  fetchReport: jest.fn(),
  updateReport: jest.fn(),
  removeReport: jest.fn(),
  setReportData: jest.fn(),
  fetchReportData: jest.fn(),
  updateReportData: jest.fn(),
  fetchReportsByState: jest.fn(),
};

export const mockReportContext = {
  ...mockReportMethods,
  report: mockReport,
  reportData: mockReportData,
  reportsByState: mockReportsByState,
  errorMessage: "",
};
```